src/main/main.js

```diff
@@ -20,14 +23,21 @@ function createWindow() {
   const menuBuilder = new MenuBuilder(win, store);
   menuBuilder.buildMenu();
 
+  const cb = () => {
+    menuBuilder.storeHasChanged();
+  };
+  storeSubscribe(cb);
+  
   win.loadURL(`file://${__dirname}/index.html`);
 
   win.on('closed', function () {
       win = null;
+      storeUnsubscribe(cb);
   });
 }
 
-app.setAccessibilitySupportEnabled(true);
```

Comment:
I wonder what exactly setAccessibilitySupportEnabled does. Electron documentation isn't great regarding this. Based on this thread (https://github.com/electron/electron/pull/15302) I wonder if keyboard issues are tied up in here. Maybe we should enable it all the time? How reliable is a screen-reader detector? Does it cover all the ATs?

Reply:
The performance penalty when enabling the Electron/Chromium's accessibility "bridge" is not insignificant (apparently).
Thus why this app framework automatically detects connected AT (via system events specific to screen readers, etc.), to switch on "a11y support" only when truly needed.

Point of interest: VisualStudio Code includes a manual switch in the user preferences, so that the default a11y toggle can be overridden.

Reply:
...and I saw that too :)

https://github.com/electron/electron/issues/16256

Reply:
https://electronjs.org/docs/tutorial/accessibility#inside-application

Reply:
Just dumping this other link here (in case we need to quickly get back to it):
https://www.chromium.org/developers/design-documents/accessibility#TOC-How-Chrome-detects-the-presence-of-Assistive-Technology

Reply:
I like the idea of putting a persisted manual override in prefs. I don't trust electron to guess right all the time.

Reply:
VisualStudio Code `editor.accessibilitySupport` = `on | off | auto`
https://code.visualstudio.com/docs/editor/accessibility#_screen-reader-mode

Reply:
Let's treat this in a future PR that addresses screen reader issues specifically (especially as we are moving to Electron5 soon, PR ready).
The default should be to *not* activate the accessibility bridge, due to performance issues. We might consider introducing a user option so that this can be force-enabled. To be discussed.